### PR TITLE
"confirm tx on main net" restyled

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-project/issues/1048

### CODE REVIEW

**Review by ryestew — COMMENTED** [review]
Right now in this PR the modal looks like this:
image

Here's a shorter line:
image

File: apps/remix-ide/src/app/ui/confirmDialog.js
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:29]: <div>You are about to create a transaction on the main network. <br>Confirm the details to send to your provider.</div>

File: apps/remix-ide/src/app/ui/confirmDialog.js
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:58]: <span>Gwei (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> for current gas price info)</span>

**Review by ryestew — CHANGES_REQUESTED** [review]

**Comment by LianaHus** [comment]
@ryestew done!
thanks

**Comment by LianaHus** [comment]
@ryestew could you please review again? TIA

**Comment by yann300** [comment]
changes sounds good, waiting for @ryestew approval

File: apps/remix-ide/src/app/ui/confirmDialog.js
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:29]: For many users, I think the word **provider** is not clear.  Is this line better: 

You are about to create a transaction on the main network. Confirm the details to send to your provider.  The provider for many users is MetaMask.  Confirming the details only sends the info to the provider -where in turn, you will need to sign the transaction to send it to the main network.

or 

You are about to create a transaction on the main network. Confirm the details to send the info to your provider.  The provider for many users is MetaMask.  The provider will ask you to sign the transaction before it is sent to the main network.

**Review by ryestew — APPROVED** [review]

**Pull request merged**
